okapi/services/caches/search/searching.inc.php

```diff
@@ -551,7 +551,33 @@ public function prepare_common_search_params()
                 }
             }
         }
-
+        
+        #
+        # powertrail_ids
+        #
+        
+        if (!is_null($tmp = $this->request->get_parameter('powertrail_ids')))
+        {
+            if (!preg_match("/^(\d+(\|\d+)*)?$/", $tmp))
+                throw new InvalidParam('powertrail_ids', "'$tmp'");
+            else {
+                if (Settings::get('OC_BRANCH') == 'oc.pl') {
+                    
+                    $extra_joins[] = 'inner join powerTrail_caches on powerTrail_caches.cacheId = caches.cache_id';
+                    #
+                    # Filter out caches from inactive powerTrails as well
+                    #
+                    $extra_joins[] = 'inner join PowerTrail on PowerTrail.id = powerTrail_caches.powerTrailId';
+                    $where_conds[] = 'PowerTrail.status = 1';
+                    
+                    $tmp = str_replace('|', ',', $tmp );
```

Comment:
Chodziło o coś takiego:

if (!is_null($tmp = $this->request->get_parameter('powertrail_ids')))
        {
            if (Settings::get('OC_BRANCH') == 'oc.pl') {
                $extra_joins[] = "inner join powerTrail_caches on powerTrail_caches.cacheId = caches.cache_id";
                $extra_joins[] = "inner join PowerTrail on PowerTrail.id = powerTrail_caches.powerTrailId";
                $where_conds[] = "PowerTrail.status = 1";
                $where_conds[] = "powerTrail_caches.powerTrailId in ('".implode(
                    "','", array_map('mysql_real_escape_string', explode("|", $tmp))
                )."')";
            } else {
                $where_conds[] = "0=1";
            }
        }